IowaPD_people.yaml

```diff
@@ -425,16 +457,42 @@ entityDefinitions:
     - !<conditions.BooleanIsNullCondition>
       column: 'OfficerID'
       reverse: true
+
+
   OthersPersonDetails:
     name: "OthersPersonDetails"
     fqn: 'ol.persondetails'
     entitySetName: 'JohnsonCounty_IowaCity_JIPeopleDetails'
     propertyDefinitions:
+      general.id:
```

Comment:
should this be the exact same pk as for the OtherPeople?

Reply:
concatenates all possible values of person pk => unique. No need to distill the MNI number here